.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      timezone: Asia/Kolkata
+      interval: weekly
+      time: "08:00"
```

Comment:
is it possible to omit the timezone and time, or can you set it to UTC timezone with 11:00 for the time

Reply:
Yes, you can set the interval/time to whatever you like.

Reply:
Updated.